common/src/models/domain/windowState.ts

```diff
@@ -0,0 +1,10 @@
+export class WindowState {
+  width?: number;
+  height?: number;
+  isMaximized?: boolean;
+  // TODO: displayBounds is an Electron.Rectangle.
+  // We need to establish some kind of client-specific global state, similiar to the way we already extend a base Account.
```

Comment:
Tech debt item for this: https://app.asana.com/0/1201461753012676/1201724590570905